### PR TITLE
NFT documentation

### PR DESCRIPTION
**Pull request by charleenfei**

### CODE REVIEW

File: docs/nfts/overview/anatomy.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+We recommend a certain anatomy for an NFT created with the Centrifuge [privacy enable ERC721 library](https://github.com/centrifuge/privacy-enabled-erc721/tree/develop).
+
+### Metadata
+
+It is useful to store publicly accessible data about the NFT in a separate metadata structure, in order to access and render this data in user facing applications. 
+
+We store this metadata in a TokenData structure which typically contains at least the document version and the address of the minter, as well as the fields which are contained in the precise proofs.
+
+For example:
```
Comment (denniswell) [inline docs/nfts/overview/anatomy.md]: For example, for an invoice, these fields could be:

File: docs/nfts/overview/anatomy.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+There are several checks that should be done prior to the minting of an NFT:
+
+1. `_latestDoc`: Checking that the document is the latest version
+
+
+2. `_checkAnchor`: Checking that the document has been properly anchored
+
+3. `_signed`: Checking that the identity trying the mint the NFT is a valid Centrifuge identity, and that the signing key used for minting contains a signing purpose and is not revoked
+
+4. `_checkTokenData`: Checking that the passed in token uniquess proof matches the data on the token to be minted
```
Comment (denniswell) [inline docs/nfts/overview/anatomy.md]: uniqueness?

**Comment by lucasvo** [comment]
@charleenfei I think it would be cool if we can have an entire implementation of an NFT with a single proof as code embedded in the documentation.

File: docs/nfts/overview/anatomy.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+### Checks
+
+There are several checks that should be done prior to the minting of an NFT:
+
+1. `_latestDoc`: Checking that the document is the latest version
+
+
+2. `_checkAnchor`: Checking that the document has been properly anchored
+
+3. `_signed`: Checking that the identity trying the mint the NFT is a valid Centrifuge identity, and that the signing key used for minting contains a signing purpose and is not revoked
```
Comment (denniswell) [inline docs/nfts/overview/anatomy.md]: ...trying **to** mint...

File: docs/nfts/overview/anatomy.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+4. `_checkTokenData`: If there is a proof present that there is no other token minted in this registry for the provided document, checking that the token data in the passed in token uniqueness proof matches the data on the token to be minted
+
+5. `verify`:  Verify the submitted proofs against the NFT to be minted. For more information, please read the information in Verification of Proof Fields
```
Comment (denniswell) [inline docs/nfts/overview/anatomy.md]: Should there be a link for "Verification of Proof Fields"?

File: docs/nfts/overview/anatomy.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+order: 2
+title: Anatomy of an NFT
+category: 1. Overview
+redirect_from:
+  - /nfts/
+---
+
+We recommend a certain anatomy for an NFT created with the Centrifuge [privacy enabled ERC721 library](https://github.com/centrifuge/privacy-enabled-erc721/tree/develop).
+
+*Please note that the proposed NFT structure will chain pending the launch of the Centrifuge Chain.*
```
Comment (denniswell) [inline docs/nfts/overview/anatomy.md]: ... structure will **change** pending...

**Pull request merged**